src/components/tabs/migrate-to-10.x.md

```diff
@@ -0,0 +1,3 @@
+### HTML
+
+Icon from [`carbon-elements`](https://github.com/IBM/carbon-elements) package is now used. Vanilla markup should be migrated to one shown in [carbondesignsystem.com](https://next.carbondesignsystem.com/components/tabs/code) site. React and other framework variants should reflect the change automatically.
```

Comment:
This one also has a few new things. Unsure if they need to be in the migration doc since they're both primarily used in the js file.

` .bx--tabs-trigger--open`, New class; Added via javascript when open in mobile view. Allows for animation.
`selectorButtonEnabled: .bx--tabs__nav-item:not(.bx--tabs__nav-item--disabled)`, Changed Selector; Allows for no-selection functionality on disabled tabs.

Reply:
Thanks @jnm2377 for pointing this out! Updated.